build.sbt

```diff
@@ -20,15 +20,23 @@ val kafkaV = "2.8.1"
 
 ThisBuild / version := "0.1-SNAPSHOT"
 
-val nussknackerV = "1.5.0"
+//val defaultNussknackerV = "1.5.0" 
+val defaultNussknackerV = "1.6.0-staging-2022-09-08-9385-369c17d35f541580d3828cf0d38bd53a2e4c7ca0-SNAPSHOT" 
```

Comment:
this will be updated on request, right?

Reply:
this is temporary. this value will be overridden on trigger from nussknacker. I'd love to have `default_nu_version = if build on staging then next snapshot else current stable` but unfortunately it is impossible because we haven't got "next snapshot" - we add suffixes with commit sha an so on.